epoxy-sample/src/main/java/com/airbnb/epoxy/SampleAdapter.java

```diff
@@ -96,11 +113,23 @@ public void onClick(View v) {
     }
   };
 
+  EditTextListener editTextListener = new EditTextListener() {
+    @Override
+    public void onTextEntered(String enteredText) {
+      Log.d(LOG_TAG, enteredText);
```

Comment:
Maybe we could make this a little more interesting and the entered text would be centered on any of the colored blocks when `add` is clicked?

If the text is empty we add no text to the blocks. If the text is longer than a certain length we show an error. I think a better example for the error would be to set the error text on the model via the adapter. We should try not to have the error logic in the model and instead just delegate all the click behavior back here.